repository/t.json

```diff
@@ -3367,6 +3367,18 @@
 				}
 			]
 		},
+		{
+		    "name": "Transpy",
+		    "details": "https://github.com/cumulus13/Transpy",
+		    "labels": ["translation", "utility"],
```

Comment:
[translate](https://packages.sublimetext.io/?q=label%3Atranslate) is actually the more common label, you can search on our labels [overview](https://packages.sublimetext.io/labels/).